priv/perf/apps/load_testing/lib/utils/account.ex

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitation
+
+defmodule LoadTesting.Utils.Account do
```

Comment:
the name of the module is a bit misleading, maybe `PrivateKey` utils is better?

Reply:
I do think `Account.new` describes better than `PrivateKey.new` though. The returned value includes `privKey`, `address`, `pubKey`